src/doc/reference.md

```diff
@@ -1114,6 +1114,16 @@ type Point = (u8, u8);
 let p: Point = (41, 68);
 ```
 
+Currently a type alias to an enum type cannot be used to qualify the
+constructors:
+
+```
+enum E { A }
+type F = E;
+let _: F = E::A;  // OK
+// let _: F = F::A;  // NG
```

Comment:
"NG"? I don't know what this means, maybe "not good"? Can we just write "Doesn't work" or something like that?